subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_poll.c

```diff
@@ -141,6 +142,11 @@ static void http_fota_dl_handler(const struct fota_download_evt *evt)
 		} else {
 			k_sem_give(&fota_download_sem);
 		}
+
+		if (ctx_ptr->status_fn) {
+			ctx_ptr->status_fn(NRF_CLOUD_FOTA_SUCCEEDED, NULL);
```

Comment:
should `k_work_cancel_delayable(&ctx_ptr->timeout_work);` be called before `status_fn` is called for all of these?

Reply:
yeah, good idea.